src/main/test/com/trivialTrivia/api/TriviaApiTest.java

```diff
@@ -1,15 +1,41 @@
 package com.trivialTrivia.api;
```

Comment:
Can we move this to match the other test structure test/java/com/trivialTrivia/api ?
I think it's causing an error when I pull this branch it- (I believe this is the cause.. if not feel free to help me out!)
![image](https://user-images.githubusercontent.com/60193714/98620474-f8edbc00-22ca-11eb-86a7-27b335680b92.png)

![image](https://user-images.githubusercontent.com/60193714/98620493-fe4b0680-22ca-11eb-908d-a69eb3d9012a.png)

Reply:
didnt even notice this, i moved the package over to the java directory